packages/eslint-plugin-svelte/src/types.ts

```diff
@@ -366,4 +366,6 @@ export interface SourceCode {
 	getCommentsAfter(nodeOrToken: NodeOrToken | AST.Token): AST.Comment[];
 
 	getCommentsInside(node: NodeOrToken): AST.Comment[];
+
+	getScope(node: NodeOrToken): Scope;
```

Comment:
our types specify that `context.getScope()` exists but it doesn't seem to in tests (it throws if we try use that)

meanwhile, `context.getSourceCode().getScope(node)` does exist

im guessing there's some compat stuff going on here to do with eslint versions removing deprecated stuff

Reply:
We need to import `getScope` from `ast-utils.js`.